### PR TITLE
Add IaaS v4 compliance check for two syseleven regions.

### PR DESCRIPTION
**Pull request by garloff**
Links on docs page and in github README to follow later.

### CODE REVIEW

File: .github/scs-compliance-check/openstack/clouds.yaml [hunk 1]
```diff
@@ -89,3 +89,19 @@ clouds:
     auth:
       auth_url: https://identity.l1a.cloudandheat.com/v3
       application_credential_id: "7ab4e3339ea04255bc131868974cfe63"
+  syseleven-dus2:
+    interface: public
+    identity_api_verion: 3
+    auth_type: "v3applicationcredential"
+    region_name: dus2
+    auth:
+      auth_url: https://identity.l1a.cloudandheat.com/v3
```
Comment (mbuechse) [inline .github/scs-compliance-check/openstack/clouds.yaml:98]:
```suggestion
      auth_url: https://keystone.cloud.syseleven.net:5000/v3
```

File: .github/scs-compliance-check/openstack/clouds.yaml [hunk 2]
```diff
@@ -89,3 +89,19 @@ clouds:
...
+      application_credential_id: s11auth
+  syseleven-ham1:
+    interface: public
+    identity_api_verion: 3
+    auth_type: "v3applicationcredential"
+    region_name: ham1
+    auth:
+      auth_url: https://identity.l1a.cloudandheat.com/v3
```
Comment (mbuechse) [inline .github/scs-compliance-check/openstack/clouds.yaml:106]:
```suggestion
      auth_url: https://keystone.cloud.syseleven.net:5000/v3
```

File: .github/workflows/check-syseleven-dus2-v4.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: "Compliance IaaS v4 of syseleveni dus2 region"
```
Comment (mbuechse) [inline .github/workflows/check-syseleven-dus2-v4.yml:1]:
```suggestion
name: "Compliance IaaS v4 of syseleven dus2 region"
```

File: .github/workflows/check-syseleven-ham1-v4.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  check-syseleven-ham1:
+    uses: ./.github/workflows/scs-compliance-check-with-application-credential.yml
+    with:
+      version: v4
+      layer: iaas
+      cloud: "syseleveni-ham1"
```
Comment (mbuechse) [inline .github/workflows/check-syseleven-ham1-v4.yml:21]:
```suggestion
      cloud: "syseleven-ham1"
```

**Review by mbuechse — CHANGES_REQUESTED** [review]

**Comment by garloff** [comment]
Thanks for the reviews and thanks for spotting my typos!

**Review by mbuechse — APPROVED** [review]

**Pull request merged**